dance/transforms/filter.py

```diff
@@ -884,3 +886,62 @@ def gini_func(x, weights=None):
     G_RSV = G_num / mean_RSV
 
     return G_RSV
+
+
+filter_genes_orders = read_conditional_parameter(path='dance/conditional_parameter.yml',
+                                                 conditional_parameter="filter_genes_orders")
+
+
+@register_preprocessor("filter", "gene")
+class FilterGenesScanpyOrder(BaseTransform):
+    """Scanpy filtering gene transformation with additional options.
+
+    Parameters
+    ----------
+    order
```

Comment:
Using lists instead of indexes is a bit more readable and doesn't interfere with automated machine learning searches

Reply:
search space in conditional_parameter.yml